class.jetpack.php

```diff
@@ -52,8 +52,8 @@ class Jetpack {
 	);
 
 	var $plugins_to_deactivate = array(
-		'stats'               => array( 'stats/stats.php', 'WordPress.com Stats' ),
-		'shortlinks'          => array( 'stats/stats.php', 'WordPress.com Stats' ),
+		'stats'               => array( 'stats/stats.php', 'Site Stats' ),
```

Comment:
The old WordPress.com Stats can't be downloaded anymore, but it's still called WordPress.com Stats:
https://wordpress.org/support/plugin/stats

Since we won't be updating that plugin to change its name, it's probably best to keep using the old name in the warnings.